lib/knigge/options.ex

```diff
@@ -352,6 +365,9 @@ defmodule Knigge.Options do
       :keyword ->
         "keyword list"
 
+      [:atom, {:list_of, :atom}] ->
+        "atom or list of atoms"
+
```

Comment:
Technically we could make this more general purpose but for now this is sufficient.

I'd only suggest to add a little reminder to future us:

```suggestion
      # For now we explicitly match on the "or" option, as soon as we add further 
      # "or"ed options this ought to be refactored to something more general purpose
      [:atom, {:list_of, :atom}] ->
        "atom or list of atoms"

```